contrib/scalapblib/src/ScalaPBWorker.scala

```diff
@@ -38,28 +30,51 @@ class ScalaPBWorker {
     }
   }
 
-  def compile(scalaPBClasspath: Agg[os.Path], protocPath: Option[String], scalaPBSources: Seq[os.Path], scalaPBOptions: String, dest: os.Path, scalaPBIncludePath: Seq[os.Path])
-             (implicit ctx: mill.api.Ctx): mill.api.Result[PathRef] = {
-    val compiler = scalaPB(scalaPBClasspath, protocPath)
-
-    def compileScalaPBDir(inputDir: os.Path) {
-      // ls throws if the path doesn't exist
-      if (inputDir.toIO.exists) {
-        os.walk(inputDir).filter(_.last.matches(".*.proto"))
-          .foreach { proto =>
-            compiler.compileScalaPB(proto.toIO, scalaPBOptions, dest.toIO, scalaPBIncludePath)
-          }
+  /** Builds the compilation arguments for scalaPBC:
+   *
+   *   - 1st seq to encapsulate the sources.
+   *   - 2nd seq for proto files.
+   *   - 3rd seq for the scalaPBC args.
+   */
```

Comment:
Instead of "1st", "2nd", ... please used proper Scaladoc `@param` attributes to make clear which parameter has which purpose.